chrome/patches/curl-impersonate.patch

```diff
@@ -3106,7 +3141,9 @@ index 8c8f43e83..030832aeb 100644
 + 
 +  /* curl-impersonate: Enable TLS extensions 18 - signed_certificate_timestamp. */
 +  // XXX: Firefox does not enable this
-+  SSL_CTX_enable_signed_cert_timestamps(backend->ctx);
++  if(data->set.ssl_enable_signed_certificate_timestamp) {
++    SSL_CTX_enable_signed_cert_timestamps(backend->ctx);
++  }
```

Comment:
Everything else was made so I could have this, btw since the comment says that Firefox does not enable this, I believe it would also be fixed, since I've follow the same defaults as `tls_session_ticket` and it turns out to be `false` for non-chrome.